.idea/misc.xml

```diff
@@ -54,4 +54,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
```

Comment:
unnecessary file

Reply:
gitignore needs updating, will deal with it next week